util/telemetry/src/otlp.rs

```diff
@@ -35,21 +35,27 @@ pub fn setup_default_tracer_with_tags(
 
     let local_hostname = hostname::get().map_err(Error::GetHostname)?;
 
-    let mut tags = vec![KeyValue::new(
-        "hostname",
-        local_hostname
-            .to_str()
-            .ok_or(Error::HostnameToString)?
-            .to_owned(),
-    )];
+    let mut tags = vec![
+        KeyValue::new("service.name", service_name.to_owned()),
+        KeyValue::new(
+            "hostname",
+            local_hostname
+                .to_str()
+                .ok_or(Error::HostnameToString)?
+                .to_owned(),
+        ),
+    ];
     for (key, value) in extra_tags.iter() {
         tags.push(KeyValue::new(*key, value.clone()));
     }
 
-    opentelemetry_jaeger::new_agent_pipeline()
-        .with_service_name(service_name)
+    let provider = opentelemetry_otlp::new_pipeline()
+        .tracing()
+        .with_exporter(opentelemetry_otlp::new_exporter().tonic())
         .with_trace_config(trace::Config::default().with_resource(Resource::new(tags)))
-        .install_simple()
-        .map_err(Error::Trace)
-        .map(Some)
+        .install_batch(opentelemetry_sdk::runtime::Tokio)
```

Comment:
This used to use `install_simple()`.
The `install_simple()` will do a full publish of a span/trace when the span closes
The batch processor will queue up finished spans locally and send them out.